administration/logging.md

```diff
@@ -172,3 +172,120 @@ log4j2.appender.ZWave.policies.size.size = 10MB
 log4j2.appender.ZWave.strategy.type = DefaultRolloverStrategy
 log4j2.appender.ZWave.strategy.max = 10
 ```
+
+## Logback Configuration File
+
+In order to define custom log patterns, log to network sockets and so on we can prepare a logging configuration file.
```

Comment:
we are using the log4j2 facade and can use whatever logging implementation. I just reintegrated sections from ESH here, they might indeed not be up to date. This PR is also not about "administration", so if someone would block the PR because of file changes here, I'd rather remove those changes.